commands/conduit_cli.py

```diff
@@ -4,11 +4,15 @@
 import time
 
 import boto3
+import botocore  # noqa
```

Comment:
What is the meaning/purpose of the `# noqa` comment?

Reply:
This comment means that linters should ignore this line.

Reply:
I need botocore to be imported in that file, so that I can mock the api call directly in one of the unit tests, since moto doesn't cover my use case